apps/openchallenges/infra/src/dns/openchallenges-dev-dns.ts

```diff
@@ -0,0 +1,23 @@
+import { Construct } from 'constructs';
+import { Route53Record } from '@cdktf/provider-aws/lib/route53-record';
+import { Alb } from '@cdktf/provider-aws/lib/alb';
+
+export class OpenChallengesDevDns extends Construct {
+  record: Route53Record;
+
+  constructor(scope: Construct, id: string, lb: Alb) {
+    super(scope, id);
+
+    // Create an alias record that points to the ALB
+    this.record = new Route53Record(this, 'record', {
+      name: 'dev.openchallenges.io',
+      zoneId: `Z08311133Q4NQKBJPU4ZX`,
```

Comment:
The zoneID could be exported from cnb-dev-dns and used here.

Reply:
This is a good id. The zone id is an output of the stack `cnb-dev`. I need to learn how to access an output from another stack.

Reply:
This improvement will be addressed in a future PR #1875.